enarx-keep-sev/src/vm.rs

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::x86_64::*;
+
+use std::error::Error;
+
+use kvm_bindings::kvm_userspace_memory_region;
```

Comment:
```suggestion
use kvm_bindings::kvm_userspace_memory_region as MemoryRegion;
```

I understand that `kvm-bindings` is auto-generated. But seeing a strangely named struct below is jarring. We can just rename it here.